src/components/snapshots/useSnapshot.ts

```diff
@@ -281,4 +283,15 @@ const copyFiles = async (snapshotFolder: string, fileRecords: ObjectRecord[] = [
   }
 }
 
+// Mutate the active config object to inject new preferences
+type Config = typeof config
+export const refreshPreferences = (config: Config) => {
+  const serverPrefs: ServerPreferences = JSON.parse(readFileSync(PREFERENCES_FILE, 'utf-8')).server
+  serverPrefKeys.forEach((key) => {
+    if (serverPrefs[key]) {
+      config[key] = serverPrefs[key] as never
```

Comment:
I really don't know why the `as never` is needed here. `config[key]` should definitely not be `never` as both `config` and the key values are from properly defined types (and config *includes* all the server pref keys)